po/pl.po

```diff
@@ -833,19 +866,19 @@ msgstr "Wystąpił nieznany błąd."
 
 #: ../src/main.cc:138
 msgid "Print information on the indicated element(s) to standard output"
-msgstr ""
+msgstr "Wydrukuj informacje wskazanym/ch pierwiastku/ach na wyjściu standardowym"
 
 #: ../src/main.cc:140 ../src/main.cc:148
 msgid "SYMBOL_OR_NUMBER"
-msgstr ""
+msgstr "SYMBOL_LUB_NUMER"
 
 #: ../src/main.cc:146
 msgid "Show the properties dialog for the indicated element(s)"
-msgstr ""
+msgstr "Pokaż okno z właściwościami na dla wskazanego/ych pierwiastka/ów"
 
 #: ../src/main.cc:154
 msgid "Display the table even if data have been printed to standard output"
-msgstr ""
+msgstr "wyświetl tabelę nawet jeśli dane zostały wydrukowane na wyjściu standardowym"
 
 #: ../src/main.cc:172 ../src/main.cc:192
 msgid "Unknown element \"%1\"."
```

Comment:
In the line below (which I can't add a comment to directly) there's an error in the existing translation -- I assume that "element" should be translated to "pierwiastek" here.